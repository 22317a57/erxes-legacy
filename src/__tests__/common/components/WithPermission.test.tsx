import { mount, shallow } from 'enzyme';
import React from 'react';

import WithPermission from '../../../modules/common/components/WithPermission';

describe('WithPermission component', () => {
  const defaultProps = {
    action: '',
    children: false
  };

  test('renders WithPermission successfully', () => {
    shallow(<WithPermission {...defaultProps} />);
  });
});
