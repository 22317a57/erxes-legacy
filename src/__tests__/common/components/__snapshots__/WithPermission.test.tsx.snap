// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithPermission component snapshot matches 1`] = `null`;
